lib/msal-core/src/Configuration.ts

```diff
@@ -36,6 +37,7 @@ export type AuthOptions = {
     clientId: string;
     authority?: string;
     validateAuthority?: boolean;
+    knownAuthorities?: Array<string>;
```

Comment:
Can we do surface changes @DarylThayil and mark it `msal@.1.3.0`? I know it isn't exact `semver` but that can help moving the below params to:
```javascript
export type AuthorityOptions = {
    validateAuthority?: boolean;
    knownAuthorities?: Array<string>;
};
```

Or even get rid of `validateAuthority` and have just `knownAuthorities` as the option in this type. That way we can default validation to true for anyone who doesn't pass `knownAuthorities`. Just a thought.